example.env

```diff
@@ -11,6 +11,8 @@ DB_PASSWORD=
 SECRET=
 RESET_SECRET=
 NODE_ENV=
+LOCAL_PORT_IBF_SERVICE=4000 # If you do not specify a port, the default port is 3000. this only applies to the override docker-compose file
```

Comment:
If we merge this PR you need to set this in your local .env @arsforza @jannisvisser